listenbrainz/db/model/pinned_recording.py

```diff
@@ -0,0 +1,69 @@
+import uuid
+
+from datetime import date, datetime, timezone, timedelta
+from pydantic import BaseModel, validator, Field
+
+
+class PinnedRecording(BaseModel):
+    """Represents a pinned recording object.
+    Args:
+        user_id: the row id of the user in the DB
+        row_id: (Optional) the row id of the pinned_recording in the DB
+        recording_mbid: the MusicBrainz ID of the recording
+        blurb_content: (Optional) the custom text content of the pinned recording
+        created: (Optional) the timestamp when the pinned recording record was inserted into DB
+        pinned_until: (Optional) the timestamp when the pinned recording is set to expire/unpin
+
+        Created is set to now() by default. Pinned_until is set to created + _daysUntilUnpin (7 days) by default.
+        If arguments are provided but either is invalid, the object will not be created.
+    """
+
+    user_id: int
+    row_id: int = None
+    recording_mbid: str
+    blurb_content: str = None
+    created: datetime = None
+    pinned_until: datetime = None
+
+    _daysUntilUnpin = 7  # default = unpin after one week
```

Comment:
Nitpick. I think we should move this constant of the model and name it like `DAYS_UNTIL_UNPIN` to be consistent with how we declare constants elsewhere.

Reply:
Done.